src/types/arinc_module.h

```diff
@@ -18,24 +18,47 @@ class ArincModule
     std::optional<name_t> moduleVersion;  /* optional */
     std::optional<APEX_INTEGER> moduleId; /* optional */
 
-    std::vector<SystemError> errors;
-    std::vector<MultiPartitionHMTable> multiPartitionHMTable; /* required */
-    std::vector<ModuleHMTable> moduleHMTable;                 /* required */
-    std::vector<PartitionHMTable> partitionHMTable;           /* required */
+    Partition partition[100];
```

Comment:
Maybe rename to partitionBuffer (or partitionMemoryBuffer), so the developers  would not be misled thinking that it's where we store the partitions